src/_data/maintainers.json

```diff
@@ -1,13 +1,6 @@
 
 {
   "all": [
-    {
```

Comment:
@cezaraugusto appreciate all the work you've done! Can you, instead of removing completely, add your info to the "legacy" array in this file? It'll make sure we still give you credit for all your work the past couple years.